plugins/hydra_ray_launcher/tests/test_ray_launcher.py

```diff
@@ -0,0 +1,75 @@
+# # Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import sys
+
+import pytest
+
+from hydra.core.plugins import Plugins
+from hydra.plugins.launcher import Launcher
+
+# This has to be included here for the LauncherTestSuite to work.
+# noinspection PyUnresolvedReferences
+# This has to be included here for the LauncherTestSuite to work.
+# noinspection PyUnresolvedReferences
+from hydra.test_utils.test_utils import (  # noqa: F401
+    TSweepRunner,
+    chdir_plugin_root,
+    sweep_runner,
+)
+from hydra_plugins.hydra_ray_launcher.ray_launcher import RayLauncher  # type: ignore
+
+# from hydra.test_utils.launcher_common_tests import (
+#     IntegrationTestSuite,
+#     LauncherTestSuite,
+# )
+
+
+chdir_plugin_root()
+
+win_msg = "Ray doesn't support Windows."
+
+
+@pytest.mark.skipif(sys.platform.startswith("win"), reason=win_msg)  # type: ignore
```

Comment:
the noxfile only test the plugin on operating systems mentioned in setup.py.
this is okay though as it would give a proper error message to someone trying to run the test manually on Windows.